### ISSUE TITLE
[FEATURE] Folders/grouping in app drawer

### ISSUE THREAD

**Issue by dgudim** [Deletescape-Media/Lawnchair#4674]
### Describe the feature

This is a reopen of #2471 

### Why would it be useful to add?

Quoting @NePoCz from the aforementioned issue:

"""
This is one of the few features I miss in the "new" Lawnchair and the only reason why I had to go back to Lawnchair 9 (or earlier), which supports folders in drawer. Any chance this will find its way back into the project? I think it is a great feature if you have many apps installed and still want to use drawer in a meaningful way and dont rely solely on desktop.
"""

### Did the feature exist in Lawnchair v2? (Play Store version)

Yes

### Example(s)

Just folders in the app drawer, I think this is self-explanatory

**Comment by tawissus** [comment]
Thank you. That's exactly what I need before I can switch. The app drawer is too confusing without tabs and folders.
Then app covers/folder icons are still missing on the home screen.

**Comment by luisotavio-dev** [comment]
I agree, this is the only feature missing for those who have a lot of apps installed.

**Comment by dgudim** [comment]
I am even willing to sponsor this. let's say, 20 euro. Is there any priority system for the features? :thinking:

**Comment by tawissus** [comment]
Good idea. I would also contribute a donation if there is an option.

**Comment by giladnoy** [comment]
Too bad this feature is missing.
It's one of the things I like the most about the Nova Launcher.

**Comment by prirai** [comment]
Same, would love this feature being implemented!

**Comment by mang0** [comment]
Yup, me too. I use Lawnchair but I never use the app drawer because it's impossible to organise. I've added homescreen folders with all the apps in, but I'd so much rather remove those additional pages and have an app drawer with folders.

**Comment by jerryn70** [comment]
Yes really missing this feature..

**Comment by connordelacruz** [comment]
Really been wanting to switch to Lawnchair, but the lack of app folders is kind of a deal breaker as simple as it is. I'd love to see this implemented

**Comment by Cris--** [comment]
The only reason to stay on Nova 8. Still no folders and/or tabs in the drawer of Lawnchair 15 possible. 😥

Anyway, Lawnchair is still the best alternative for me.

**Comment by Xxeenon** [comment]
Really want to switch from Nova but the inability to add folders in Lawnchair app drawer is kind of a deal breaker for me.

**Comment by MrSluffy** [comment]
done : https://github.com/LawnchairLauncher/lawnchair/pull/5189
